### PR TITLE
feat(slack): Set level to color on slack attachment

### PR DESCRIPTION
**Pull request by iiogmgo**
hello, 
I want to see again the color of level on slack notification like before sentry integration(web hook).

Like this
webhook-screenshot

### CODE REVIEW

**Comment by EvanPurkhiser** [comment]
@saragilford have any product thoughts on if we would want to merge this?

**Comment by saragilford** [comment]
From a product perspective, this works for me.

File: src/sentry/integrations/slack/utils.py [hunk 1]
```diff
@@ -18,8 +18,15 @@
 logger = logging.getLogger('sentry.integrations.slack')
 
 # Attachment colors used for issues with no actions take
-NEW_ISSUE_COLOR = '#E03E2F'
```
Comment (EvanPurkhiser) [inline src/sentry/integrations/slack/utils.py]: Could you get rid of `NEW_ISSUE_COLOR` and then down where we set the color just default to `LEVE_TO_COLOR['error']` if there is no level.

**Review by evanpurkhiser — CHANGES_REQUESTED** [review]
Just one minor comment @iiogmgo and then this is good to go :)

**Review by evanpurkhiser — APPROVED** [review]

**Comment by EvanPurkhiser** [comment]
Thanks so much @iiogmgo :D

**Pull request merged**